### PR TITLE
207 organisation replacement

### PR DESCRIPTION
**Pull request by JacqueGM**
Closes https://github.com/CirclesUBI/circles-core/issues/207

It also includes 
 - updating all the  test and files related to include the work from relayer-replacement branch that Juanen did previously

### CODE REVIEW

**Review by juanenrisley — CHANGES_REQUESTED** [review]
The jsdocs should be review before merging. All comments for the organization module methods must be updated.

File: src/organization.js [hunk 1]
```diff
@@ -2,288 +2,235 @@ import CoreError, { ErrorCodes } from '~/common/error';
...
 
-      const txHash = await utils.executeTokenSafeTx(account, {
-        safeAddress: options.from,
+    await safe.sendTransaction(account, {
+      safeAddress: options.to,
+      transactionData: {
         to: hub.options.address,
-        txData,
-      });
+        data: hub.methods.trust(options.from, 100).encodeABI(),
```
[the remark this answers is not in the document]
Comment (JacqueGM) [reply src/organization.js]: done

**Review by juanenrisley — APPROVED** [review]
👍🏿 Alrighty!

**Review by juanenrisley — APPROVED** [review]
👍🏿 <3

**Review by juanenrisley — APPROVED** [review]
👍🏿

**Pull request merged**